docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -2,234 +2,205 @@
-              "buckets" : [
-                {
-                  "key" : "41893910-6bca-11eb-9e0d-85d233e3ee35",
-                  "doc_count" : 6
-                }
-              ]
-            }
-          }
-        }
-      ]
-    }
+    "task" : {
+      "schedule" : {
+        "interval" : "5s"
+      },
+      "taskType" : "alerting:.index-threshold",
+      "retryAt" : null,
+      "runAt" : "2021-05-10T05:18:02.704Z",
+      "scope" : [
+        "alerting"
+      ],
+      "startedAt" : null,
+      "state" : """{"alertInstances":{},"previousStartedAt":"2021-05-10T05:17:45.671Z"}""",
+      "params" : """{"alertId":"30d856c0-b14b-11eb-9a7c-9df284da9f99","spaceId":"default"}""",
+      "ownerId" : null,
+      "scheduledAt" : "2021-05-10T04:50:07.333Z",
+      "attempts" : 0,
+      "status" : "idle"
+    },
+    "references" : [ ],
+    "updated_at" : "2021-05-10T05:17:58.000Z",
+    "coreMigrationVersion" : "8.0.0",
+    "type" : "task"
   }
 }
 --------------------------------------------------
-<1> Most rule execution durations fall within the first bucket (0 - 1 seconds).
-<2> A single rule with id `41893910-6bca-11eb-9e0d-85d233e3ee35` took between 30 and 31 seconds to execute.
 
-Use the <<get-rule-api,Get Rule API>> to retrieve additional information about rules that take a long time to execute.
\ No newline at end of file
+What you can see above is the Task that backs the rule, and for the rule to work, this task must be in a healthy state. This information is available via <<task-manager-api-health, health API>> or via verbose logs if debug logging is enabled.
+When diagnosing the health state of the task, you will most likely be interested in the following fields:
+
+`status`:: This is the tasks’s current status - is it running, meaning Task Manager is currently running it; or is it idle, meaning we’re waiting for Task Manager to run it at some point in the future; or is it failed, meaning Task Manager has tried to run it and failed?
+`runAt`:: This is when the task is scheduled to run next. If this is in the past and the status is idle, this means Task Manager has fallen behind or isn’t in fact running at all. If it’s in the past but the status is running, then that means Task Manager has picked it up and is working on it as we speak, which is fine and considered healthy.
+`retryAt`:: Another time field, like runAt. If this field is populated, it means that Task Manager is currently running the task and if, for whether reason, the task doesn’t complete (and hasn’t been marked as failed) then Task Manager will give it another attempt at the time specified under retryAt.
+
+Investigating the underlying Task can help you gauge whether the problem you’re seeing is rooted in the Alert not running at all, whether it’s running and failing or whether it is running, but exhibiting behaviour that is different than what was expected (at which point you should focus on the Rule itself, rather than the Task).
```

Comment:
```suggestion
Investigating the underlying task can help you gauge whether the problem you’re seeing is rooted in the alert not running at all, whether it’s running and failing, or whether it is running, but exhibiting behavior that is different than what was expected (at which point you should focus on the rule itself, rather than the task).
```